src/api/dependency-graph/src/router.js

```diff
@@ -0,0 +1,13 @@
+const { Router } = require('@senecacdot/satellite');
+
+module.exports = function (graphConstructor) {
+  const graph = graphConstructor();
+
+  const router = Router();
+
+  router.get('/project/:id', async (req, res) => {
+    res.status(200).json(graph.getProject(req.params.id));
```

Comment:
Let's add `cache-control` headers for this, since it won't update very often.  We can probably cache for 30 mins or an hour?

Reply:
You also need to deal with error cases.  If `getProject` throws, your server will crash.